support/doc/development/tests.md

```diff
@@ -2,6 +2,12 @@
 
 ## Preparation
 
+If you not already installed devDependencies:
```

Comment:
I think you should point to the contributing guide instead, because you also need to build peertube

Reply:
Building Peertube is 2 steps under :)

Reply:
Oup's,  then remove `--production=false`

Reply:
As said in comments to the previous point, launching unit tests with `TS_NODE_TRANSPILE_ONLY=true mocha -- --timeout 30000 --exit -r ts-node/register -r tsconfig-paths/register --bail server/tests/api/videos/video-transcoder.ts` doesn't work for me without --production=false